auth-web/package-lock.json

```diff
@@ -58,7 +58,7 @@
         "@types/vuelidate": "^0.7.13",
         "@typescript-eslint/eslint-plugin": "^6.4.0",
         "@typescript-eslint/parser": "^6.4.0",
-        "@volar-plugins/vetur": "*",
+        "@volar-plugins/vetur": "latest",
```

Comment:
Please undo this change.

Also, please update the app version again.

Then, go ahead and merge this PR.

Reply:
Thanks Sev! I just undo this and updated the app version again.
Could you please merge it?
Thank you.